### PR TITLE
Working cors examples

### PR DESCRIPTION
**Pull request by robertohuertasm**
I've added the `options` endpoint in order to enable preflight requests and also created a new example using manual setting of the headers to avoid issue mentioned in #365 & #324.

### CODE REVIEW

File: examples/enable_cors_header_set_issue.rs [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#[macro_use] extern crate nickel;
+extern crate hyper;
```
Comment (jolhoeft) [inline examples/enable_cors_header_set_issue.rs]: In response to #368, we are rexporting hyper, so I think you should be able to remove this line.

**Comment by jolhoeft** [comment]
Sorry, I somehow missed this. I have one minor comment.

File: examples/enable_cors_header_set_issue.rs (hunk 1 above)
Comment (robertohuertasm) [reply examples/enable_cors_header_set_issue.rs]: Ok, I just copied it from the previous example. Should I remove it there too?

**Comment by GitCop** [comment]
Thanks for contributing! Unfortunately, I'm here to tell you there were style issues with your Pull Request:

 * Commit: bf819e783efdccf735dfd831b47618276d44e629
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/nickel-org/nickel.rs/blob/master/contributing.md

---
This message was auto-generated by https://gitcop.com

**Comment by jolhoeft** [comment]
Thanks

**Pull request merged**